docs/github.md

```diff
@@ -112,6 +129,18 @@ $ # Push the updated branch to the remote. Jujutsu automatically makes it a forc
 $ jj git push --branch your-feature
 ```
 
+## Working with other people's branches
+
+By default `jj git clone` and `jj git fetch` clone all active branches from
+the remote, this means that if you want to iterate or test another 
+contributors branch you can either `jj new <branchname>` or 
+`jj edit <branchname>`.
```

Comment:
As above, I think it's better to recommend only the safer workflow and drop this line

Reply:
Removed the `jj edit` part.